src/main/webapp/feed.html

```diff
@@ -3,11 +3,83 @@
   <head>
     <meta charset="UTF-8">
     <title>Friends of Friends</title>
+     <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
```

Comment:
nit: same as last CL, remove "latest" or comments altogether

Can you think of a good way to share common elements like this across pages without duplicating code?

Reply:
I actually brought this up with Nik today, we found a JS library that looks useful:

https://github.com/LexmarkWeb/csi.js

Perhaps worth trying out for the common layout elements (e.g. navigation) in a follow-up PR; you can probably just leave `<script>` and `<link>` tags as-is though.

Reply:
ok! I will add this in a follow up once these get merged in